flocker/apiclient/test/test_client.py

```diff
@@ -325,7 +325,7 @@ def test_acquire_lease_conflict(self):
             acquire a lease that is held by another node.
             """
             dataset_id = uuid4()
-            d = self.client.acquire_lease(dataset_id, self.node_1, 10)
+            d = self.client.acquire_lease(dataset_id, self.node_1, 60)
```

Comment:
:question: You could still check the result of `acquire_lease` to check the returned lease matches the spec, but really that is a separate test.